plugins/inputs/turbostat/plugin.go

```diff
@@ -0,0 +1,87 @@
+package turbostat
+
+import (
+	"context"
+	"errors"
+	"os/exec"
+	"sync"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type Turbostat struct {
+	Command []string `toml:"command"`
+
+	Log telegraf.Logger
+
+	cancel context.CancelFunc
+}
+
+func (*Turbostat) Gather(telegraf.Accumulator) error {
+	return nil
+}
+
+func (*Turbostat) SampleConfig() string {
+	return ""
+}
```

Comment:
You should add:
```
//go:embed sample.conf
var sampleConfig string
```
to plugin and return `sampleConfig ` here.

Please also remember about `//go:generate ../../../tools/readme_config_includer/generator` above the package name.
Take a look at how this is done in other plugins.

Reply:
Done.